src/signal.ts

```diff
@@ -1,10 +1,20 @@
-import {Listener, WritableSignal} from './interfaces';
-
 import {ExtendedSignal} from './extended-signal';
-import { ReadableSignal } from './index';
+import {ReadableSignal} from './index';
+import {Listener, WritableSignal} from './interfaces';
 
 export class Signal<T> extends ExtendedSignal<T> implements WritableSignal<T>, ReadableSignal<T> {
+    public static setDefaultListener(listener: Listener<any>) {
+        this._staticDefaultListener = listener;
```

Comment:
I don't know why, but in cases like this I usually do Signal._staticDefaultListener. Either works, the difference would be if the context were not set. I guess it doesn't really matter, just never thought to do it this way for some reason :-)